example-get-started-experiments/code/README.md

```diff
@@ -107,47 +103,3 @@ This tag also contains a GitHub Actions workflow that reruns the pipeline if any
  changes are introduced to the pipeline-related files. 
 [CML](https://cml.dev/) is used in this workflow to provision a cloud-based GPU 
 machine as well as report model performance results in Pull Requests.
-
-## Deploying the model
```

Comment:
Removing this section for now.
Will properly close https://github.com/iterative/example-repos-dev/issues/230 on a subsequent P.R. adding the sagemaker code